bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx66.html

```diff
@@ -0,0 +1,182 @@
+                entrypoint='whatsnew',
+                button_class='mzp-c-button',
+                utm_source='whatsnew',
+                utm_params={'campaign': 'fxa-embedded-form', 'content': 'whatsnew', 'medium': 'referral'})
+            }}
+          </div>
+        </div>
+      </section>
+
+      {% if show_newsletter %}
+      <section class="c-sticky-signup newsletter-form-cta">
+        <div class="mzp-l-content">
+          <div class="c-sticky-signup-container">
+            <div class="c-sticky-signup-content">
+              <h2 class="c-sticky-signup-title">{{ _('Get tips to live your best online life right in your inbox.') }}</h2>
+              <button type="button" class="sticky-dismiss">Dismiss this prompt.</button>
+            </div>
+            <div class="mzp-c-newsletter">
+              {{ email_newsletter_form(
+                protocol_component=True,
+                include_title=False,
+                spinner_color='#fff',
+                button_class='mzp-c-button')
+              }}
+            </div>
+          </div>
+        </div>
+      </section>
+      {% endif %}
+
+    {% set content_blocking_image='firefox/whatsnew_64/hero.png' if DIR == 'ltr' else 'firefox/whatsnew_64/hero-rtl.png' %}
+    {% call hero(
+     title=_('One log-in. Power and security everywhere.'),
+     class='mzp-has-image mzp-l-reverse t-wn64 show-fxa-default show-fxa-supported-signed-out',
+     image_url=content_blocking_image,
+     include_highres_image=true,
+     include_cta=True
+    ) %}
+    <p class="show-fxa-supported-signed-out show-fxa-default">
+      <a href="https://accounts.firefox.com/signup?service=sync&context=fx_desktop_v3&entrypoint=mozilla.org-fx.wnp64&utm_content=%2Ffirefox%2FWNP%2F64%2F&utm_campaign=wnp64&utm_source=wnp-64" class="mzp-c-button mzp-t-download">{{_('Create a Firefox Account') }}</a><br>
+      <span class="wn64-signin">{{ _('Have an account?') }} <a href="https://accounts.firefox.com/?action=email&service=sync&context=fx_desktop_v3&entrypoint=mozilla.org-fx.wnp64&utm_content=%2Ffirefox%2FWNP%2F64%2F&utm_campaign=wnp64&utm_source=wnp-64">{{ _('Log in') }}</a></span>
+    </p>
+    {% endcall %}
+    {% call hero(
+     title=_('It shouldn’t be hard to own your life online.'),
+     desc=_('That’s why Firefox gives more power to you with every update.'),
+     class='mzp-has-image mzp-l-reverse t-wn64 show-fxa-supported-signed-in',
+     image_url=content_blocking_image,
+     include_highres_image=true
+    ) %}
+    {% endcall %}
+
+    <div class="mzp-l-content">
+      <div class="wn66-benefit wn66-send">
+        {{ high_res_img('firefox/whatsnew_66/fxa-send.png', {'alt': '', 'width': '590', 'height': '590'}) }}
+        <h3>{{_('Take total control of the files you share ')}}</h3>
+        <p>{{_('Feel like a secret agent when you send files online, with a private, encrypted link that ‘self-destructs.’')}}</p>
+        <p><a class="mzp-c-cta-link" data-app="send" href="
```

Comment:
Is there a reason why this is a cta-link, as opposed to the rest that are styled like a button?

Reply:
Oh I see, it doesn't open a modal 👍

Reply:
Consulted with designers to see if we wanted to do something custom to make it seem less out of place. They decided to keep it protocol.